### PR TITLE
Change navbar color to red

### PR DESCRIPTION
**Pull request by andrerferrer**
1- What you have done
  changed the color of the navbar from white to red

2- How to test it
  go to the homepage and the navbar color should be red

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .navbar-lewagon {
   justify-content: space-between;
-  background: white;
+  background: red;
```
Comment (tonipanacek) [inline app/assets/stylesheets/components/_navbar.scss]: are we sure we want to use this red here?

Comment (andrerferrer) [inline app/assets/stylesheets/components/_navbar.scss]: use this color instead #FD1015

It will look less ugly

**Comment by Georgy5** [comment]
The Logo goes under in the red background color and the Login doesn't look good. I think we should use a different color for the navbar background.

**Comment by Georgy5** [comment]
Acceptable

**Comment by andrerferrer** [comment]
@tonipanacek @Georgy5 
please check it again with the new changes

**Pull request merged**